specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7197,6 +7292,51 @@
       },
       "description": "A skill to split a string into chunks of text."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchIndexerSkill"
+        }
+      ],
+      "properties": {
+        "defaultLanguageCode": {
```

Comment:
Based on other skills, this should be `"x-nullable": true` since you send the others will a `null` value, or will that not be the case here? Better not to waste bandwidth with `null` values if there's no benefit.